index.html

```diff
@@ -10,23 +10,30 @@
     <link rel="icon" href="images/favicon.ico">
 </head>
 
-<body id="preview" data-spy="scroll" data-target="#menu-dropdown">
+<body
```

Comment:
tutaj nie musicie stosować tego typu zapisu, pamiętajcie jednak o nim gdy będziecie używać reacta i jsx

Reply:
co masz na myśłi mówiąć "tego"?

Reply:
zmiana obejmowała przepisania atrybutów do osobnych linii i o tym była mowa

Reply:
a mogę to zostawić skoro będzie to naszą dobrą praktyką "in future"?